actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -1691,6 +1693,17 @@ private void createUser(Request actorMessage) {
     response.put(JsonKey.ACCESSTOKEN, accessToken);
     sender().tell(response, self());
 
+    /*
+     * Create User Entity in Registry
+     */
+    if(ProjectUtil.getConfigValue("sunbird_open_saber_bridge_enable").equalsIgnoreCase("true")){
+		 String firstName = (String) actorMessage.getRequest().get(JsonKey.FIRST_NAME);
+		 Map<String,Object> extensionMap = new HashMap<String,Object>();
```

Comment:
Instead of creating a new hashmap, can we pass the received create user request map as it is?

Reply:
Review Comment Incorporated in Code